code/components/citizen-server-impl/include/state/SyncTrees_RDR3.h

```diff
@@ -367,7 +400,22 @@ struct CDoorScriptGameStateDataNode { bool Parse(SyncParseState& state) { return
 struct CHeliHealthDataNode { bool Parse(SyncParseState& state) { return true; } };
 struct CHeliControlDataNode { bool Parse(SyncParseState& state) { return true; } };
 
-struct CObjectCreationDataNode { bool Parse(SyncParseState& state) { return true; } };
+struct CObjectCreationDataNode
+{
+	uint32_t m_model;
+
+	bool Unparse(SyncUnparseState& state)
+	{
+		state.buffer.Write<uint32_t>(32, m_model);
```

Comment:
Why would you implement unfinished `Unparse`? And why not implement `Parse` then?
Also, are you sure that this is 32 bits of model? That's not so obvious according to data node's code. It's most likely a single bit, if this bit is true, it read something with a 32 bits size.